pom.xml

```diff
@@ -293,7 +294,6 @@
         <dependency>
             <groupId>org.scijava</groupId>
             <artifactId>scijava-common</artifactId>
-            <version>2.61.0</version>
```

Comment:
No need to specify versions that are managed in the parent POM. You can leave away the `<version>` tag entirely.